docs/source/advanced_topics/11_memory_representation_function.rst

```diff
@@ -0,0 +1,50 @@
+Conversions Between GPU Memory Formats
+---------------------------------------------
```

Comment:
Please, mention that CuPy and PyTorch are compatible and thus we do not provide functions for conversion between them. Add a link to the relevant documentation: https://docs.cupy.dev/en/stable/user_guide/interoperability.html#pytorch

Reply:
I've added examples and a link to the necessary functions from PyTorch and CuPy in the documentation